daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md

```diff
@@ -0,0 +1,605 @@
+---
+type: docs
+title: "Quickstart: Publish and Subscribe"
+linkTitle: "Publish and Subscribe"
+weight: 70
+description: "Get started with Dapr's Publish and Subscribe building block"
+---
+
+Let's take a look at Dapr's [Publish and Subscribe (Pub/sub) building block]({{< ref pubsub >}}). In this quickstart, you will set up a publisher microservice and a subscriber microservice to demonstrate how Dapr enables a Pub/sub pattern.
+
+1. Using a publisher service, developers can repeatedly publish messages to a topic.
+1. [A Pub/sub component](https://docs.dapr.io/concepts/components-concept/#pubsub-brokers) queues or brokers those messages (using Redis Streams, RabbitMQ, Kafka, etc.).
+1. The subscriber to that topic pulls messages from the queue and processes them.
+
+Select your preferred language before proceeding with the quickstart.
+
+{{< tabs "Python" ".NET" "JavaScript" >}}
+ <!-- Python -->
+{{% codetab %}}
+
+### Pre-requisites
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+
```

Comment:
For all quickstarts, Docker Desktop is also a pre-req since we're relying on redis for most examples